manifests/charts/istio-cni/templates/daemonset.yaml

```diff
@@ -20,15 +20,15 @@ metadata:
 spec:
   selector:
     matchLabels:
-      k8s-app: istio-cni-node
+      k8s-app: istio-cni-node{{- if not (eq .Values.revision "") }}-{{ .Values.revision }}{{- end }}
   updateStrategy:
     type: RollingUpdate
     rollingUpdate:
       maxUnavailable: {{ .Values.cni.rollingMaxUnavailable }}
   template:
     metadata:
       labels:
-        k8s-app: istio-cni-node
+        k8s-app: istio-cni-node{{- if not (eq .Values.revision "") }}-{{ .Values.revision }}{{- end }}
```

Comment:
usually revision is added as a separate label. see the label in the daemonset (line 17 in this file)

Reply:
The revision is appended here to provide unique `selector.matchLabels` for pods created by each install-cni DS instance.